src/elements/content-sidebar/SidebarNav.js

```diff
@@ -100,6 +104,15 @@ const SidebarNav = ({
                             <IconMetadataThick />
                         </SidebarNavButton>
                     )}
+                    {isDocgenTemplate && (
+                        <SidebarNavButton
+                            data-resin-target={SIDEBAR_NAV_TARGETS.DOCGEN}
+                            sidebarView={SIDEBAR_DOCGEN}
+                            tooltip="Box Doc Gen"
+                        >
+                            <Tag16 className="bcs-SidebarNavSignButton-icon" />
```

Comment:
the class name was intended for the Sign button and doesn't follow SUIT naming for this file. should there be a general class created?